rc4me/run.py

```diff
@@ -96,5 +97,25 @@ def reset(ctx: Dict[str, RcDirs]):
     rc_dirs.link_files()
 
 
+@cli.command()
+@click.pass_context
+def select(ctx: Dict[str, RcDirs]):
+    """Swap rc4me configurations
+
+    Displays all available repos and allow user to select one
+    """
+    # Init rc4me directory variables
+    rc_dirs = ctx.obj["rc_dirs"]
+    my_repos = rc_dirs.get_rc_repos()
+    # Show all dirs that aren't curr/prev
+    title = "Please select the run commands repo you want to use: "
```

Comment:
This command is a bit confusing: maybe "Please select the repo/configuration you want to use:" ?

Reply:
👍